client/src/pages/Profil.js

```diff
@@ -1,9 +1,114 @@
-import React from "react";
+import React, { useReducer, useState } from "react";
+import styled from "styled-components";
+import FileUpload from "../components/UploadComponent";
 
 export default function Profil() {
+  const [submitting, setSubmitting] = useState(false);
+  const [newUserInfo, setNewUserInfo] = useState({
+    profileImages: [],
+  });
+
+  const formReducer = (state, event) => {
+    if (event.reset) {
+      return {
+        name: "",
+        status: "",
+      };
+    }
+    return {
+      ...state,
+      [event.name]: event.value,
+    };
+  };
+
+  const [formData, setFormData] = useReducer(formReducer, {});
+
+  const updateUploadedFiles = (files) =>
+    setNewUserInfo({ ...newUserInfo, profileImages: files });
+
+  const handleSubmit = (event) => {
```

Comment:
I've noticed that you called the `event` only `e` in another file. Be consistent when using this. So either always write `event` or always use `e`